gulpfile.js/import/importRoadmap.js

```diff
@@ -98,33 +127,50 @@ async function importRoadmap() {
       }
 
       roadmap.push({
-        'wg_name': workingGroupName,
+        'wg_slug': workingGroupSlug,
+        'wg_name': meta.title,
         'created_at': createdAt,
-        'status_update': statusUpdate || '',
-        'quarter': quarter || '',
-        'title': title,
+        'status_update': statusUpdate,
+        'quarter': quarter,
         'number': issue.number,
-        'author': issue.user.login,
         'html_url': issue.html_url,
-        'body': issue.body,
+        'body': body,
       });
     }
 
     log.info(`.. ${wg.name} - ${issues.length} issues imported`);
   }
 
-  // Sort issues by date
+  // Get full name from working group meta.yaml and predefined color based on roadmap config
+  workingGroups = [...new Set(workingGroups.sort())];
+  workingGroups = workingGroups.map((group, index) => {
+    return {
+      'slug': group.slug,
+      'name': group.name,
+      'color': config.workingGroupColors[index] || '#000',
```

Comment:
I guess it would make more sense to define the colors in the config in an object (`{"wg-outreach": "red"}`) instead of an array.

Otherwise, the colors would shift as soon as a new working group gets added, no?

Also either put `#000` in a constant or in the configuration file.